src/main/java/microsoft/exchange/webservices/data/autodiscover/AlternateMailbox.java

```diff
@@ -164,8 +177,44 @@ public String getServer() {
    *
    * @param server the new server
    */
-  protected void setServer(String server) {
+  protected void setServer(final String server) {
     this.server = server;
   }
 
+  /**
+   * Gets the SMTP address.
+   *
+   * @return
+   */
+  public String getSmtpAddress() {
```

Comment:
Let's add useful documentation about smtp address and owner smtp address, defining whether they can be null, what combinations are valid based on alternate mailbox type.  Otherwise, the API user won't know what to expect.